### PR TITLE
User avatar

### PR DESCRIPTION
**Pull request by bwencke**

### CODE REVIEW

**Comment by bwencke** [comment]
Probably want to actually get user data before merging this.

File: imports/ui/pages/users/UserPage.jsx
Comment (garr741) [inline imports/ui/pages/users/UserPage.jsx]: Uhh no.

**Review by garr741 — CHANGES_REQUESTED** [review]

**Review by garr741 — APPROVED** [review]

File: imports/ui/pages/users/UserPage.jsx
Comment (ashleyzeg) [inline imports/ui/pages/users/UserPage.jsx]: single quotes?

**Review by ashleyzeg — APPROVED** [review]

**Comment by bwencke** [comment]
@garr741 @ashleyzeg @mhoc I changed this PR a lot since you guys approved it, so if you want to look at it again, that would be awesome. Also, "UserCard" will be moved once Ashley finished the about me and skills components.

**Review by ashleyzeg — APPROVED** [review]

**Pull request merged**
